src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_mysql.py

```diff
@@ -331,7 +355,11 @@ def flexible_server_update_custom_func(cmd, client, instance,
                               standby_availability_zone=standby_availability_zone,
                               auto_grow=auto_grow,
                               replication_role=replication_role,
-                              instance=instance)
+                              instance=instance,
+                              byok_identity=byok_identity,
+                              backup_byok_identity=backup_byok_identity,
```

Comment:
We are not allowing this for backup iteration? Or Cong said to open this parameter open for customer? (Only not exposing georestore, restore coammdns with backup parameter and leaving these params)

Reply:
I have a check that makes the command fail if the server we are creating or updating has geo-redundant-backup enabled and data encryption.

Reply:
Now we are allowing data encryption with geo-redundant servers.